.circleci/config.yml

```diff
@@ -148,6 +151,35 @@ jobs:
           paths:
             - udata_front/theme/gouvfr/static
 
+
+  accessibility:
+    docker:
+      - image: cimg/python:3.7.13-browsers
+      - image: circleci/mongo:3.6-stretch-ram
+      - image: redis:alpine
+    steps:
+      - checkout
+      - browser-tools/install-chrome
+      - attach_workspace:
+          at: .
+      - run:
+          name: Generate udata fixtures
+          command: |
+            pip install -e .
```

Comment:
We reinstall package because we can't use venv since we're on a next-gen CircleCI image using pyenv.